trigger/payloads.go

```diff
@@ -465,8 +465,12 @@ func buildLink(uiBase string, buildID string) string {
 	return fmt.Sprintf("%s/build/%s/", uiBase, url.PathEscape(buildID))
 }
 
-func versionLink(uiBase string, versionID string) string {
-	return fmt.Sprintf("%s/version/%s/", uiBase, url.PathEscape(versionID))
+func versionLink(uiBase string, versionID string, redirectSpruceUsers bool) string {
```

Comment:
I think all callers of this should be passing true, so we don't need the extra parameter. The build and task links also need similar handling for redirecting to spruce (build should just be the version page filtered to that variant)

Reply:
Can you add the same handling for the build and task pages?

Reply:
done!